app/models/barbeque/job_queue.rb

```diff
@@ -16,4 +18,44 @@ class Barbeque::JobQueue < Barbeque::ApplicationRecord
   def sqs_queue_name
     SQS_NAME_PREFIX + name
   end
+
+  def update_policy!
+    client = Aws::SQS::Client.new
+    attrs = client.get_queue_attributes(
```

Comment:
Umm, I don't think calling HTTP API is not model's responsibility. I'd do that in service class™️ or simply in controller.